tests/tests_pytorch/loggers/test_logger.py

```diff
@@ -257,6 +257,18 @@ def test_dummylogger_noop_method_calls():
     logger.log_metrics("1", 2, three="three")
 
 
+def test_dummlogger_arbitrary_method_calls(caplog):
+    """Test that the DummyLogger can be called with non existing methods."""
+    logger = DummyLogger()
+    # Example method from WandbLogger
+    logger.log_text("1", columns=["1", "2", "3"], data=[[1, 2, 3]])
+
+    assert (
+        "The `DummyLogger` is being used and the method `log_text` was called on it "
+        "but it does not exist. This is the case when `fast_dev_run` is enabled\n" in caplog.text
+    )
```

Comment:
```suggestion
    with caplog.at_level(level=logging.INFO):
        logger.log_text("1")

    assert "method `log_text` was called on it but it does not exist" in caplog.text
```
do import `logging`

Reply:
Capturing logging is not required as the dummy method is not calling `rank_zero_info` anymore